src/Tribe/Views/V2/Hooks.php

```diff
@@ -282,9 +283,28 @@ public function filter_excerpt_length( $length ) {
 	 *
 	 * @param string $link The excerpt read more link.
 	 *
-	 * @return int The modified excerpt read more link, if required.
+	 * @return string The modified excerpt read more link, if required.
 	 */
 	public function filter_excerpt_more( $link ) {
 		return $this->container->make( Template\Excerpt::class )->maybe_filter_excerpt_more( $link );
 	}
+
+	/**
+	 * Filters the `admin_post_thumbnail_html` to add image aspect ratio recommendation.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $html The HTML for the featured image box.
+	 *
+	 * @return string The modified html, if required.
+	 */
+	public function filter_admin_post_thumbnail_html( $html ) {
+
+		if ( TEC::POSTTYPE !== get_current_screen()->post_type ) {
+			return $html;
+		}
+
+		return $html . __( 'We recommend a 16:9 aspect ratio for featured images.', 'the-events-calendar' );
```

Comment:
only suggestion here is to wrap this in a `<p>` tag with classes `hide-if-no-js howto`